<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewRegistration</class>
 <widget class="QWidget" name="ViewRegistration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1127</width>
    <height>802</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#RegistrationWindow {
	
	background-color: rgb(255, 255, 255);


}

QLineEdit {
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);
	border-color: rgb(112, 129, 137);
	font: 12pt &quot;Segoe UI&quot;;
}

QPushButton {
    padding: 5px 10px;
    border-radius: 5px;
    outline: none;
    color: rgb(0, 0, 0);
    font: 700 9pt &quot;Segoe UI&quot;;
    background-color: rgb(192, 220, 161);
    border: 2px solid rgb(145, 185, 120);
}

QPushButton:hover {
    background-color: rgb(161, 184, 135); 
    border: 2px solid rgb(130, 165, 110); 
}

QPushButton:pressed {
    background-color: rgb(145, 165, 120);
    border: 2px solid rgb(130, 155, 100); 
    padding-top: 10px; 
    padding-bottom: 6px; 
}


QLabel{
	font: 12pt &quot;Segoe UI&quot;;
	color: rgb(0, 0, 0);
}

QTextBrowser {
	background-color: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContents</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
     </property>
     <widget class="QWidget" name="registrationScroll">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-52</y>
        <width>1111</width>
        <height>863</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LayoutDirection::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">#registrationScroll {
	background-color: rgb(255, 255, 255);
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,0,1">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="bannerBackground" native="true">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>150</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">#bannerBackground {
	background-image: url(:/branding/images/brand/background.jpg);
}</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
          <item>
           <widget class="QLabel" name="bannerLogo">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="images.qrc">:/branding/images/brand/logoSmall.png</pixmap>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="bannerTitle">
            <property name="styleSheet">
             <string notr="true">#bannerTitle {
	color: rgb(255, 255, 255);
	font: 700 60pt &quot;Segoe UI&quot;;
}</string>
            </property>
            <property name="text">
             <string>City Library</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="pageContent" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>150</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>150</number>
          </property>
          <property name="verticalSpacing">
           <number>40</number>
          </property>
          <item row="1" column="0">
           <widget class="QWidget" name="formContainer" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>375</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">#formContainer {
	border-width: 2px;
	border-style: solid;
	border-color: rgb(176, 209, 152);
	background-color: rgb(227, 238, 219);
}</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,0,0,0">
             <property name="sizeConstraint">
              <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="formTitle">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>65</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">#formTitle {
	border-width: 2px;
	border-style: solid;
	border-color: rgb(176, 209, 152);
	font: 700 12pt &quot;Segoe UI&quot;;
	color: rgb(0, 0, 0);
	background-color: rgb(215, 236, 199);
}</string>
               </property>
               <property name="text">
                <string>   Sign Up Form</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="registerFormLayout" native="true">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>250</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QLabel{
	font: 700 10pt &quot;Segoe UI&quot;;
}</string>
               </property>
               <layout class="QFormLayout" name="formLayout">
                <property name="horizontalSpacing">
                 <number>15</number>
                </property>
                <property name="leftMargin">
                 <number>25</number>
                </property>
                <property name="rightMargin">
                 <number>25</number>
                </property>
                <item row="0" column="0">
                 <widget class="QLabel" name="nameLabel">
                  <property name="text">
                   <string>Name</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLineEdit" name="nameLineEdit"/>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="usernameLabel">
                  <property name="text">
                   <string>Username</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLineEdit" name="usernameLineEdit"/>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="passwordLabel">
                  <property name="text">
                   <string>Password</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QLineEdit" name="passwordLineEdit"/>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="phoneLabel">
                  <property name="text">
                   <string>Phone</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QLineEdit" name="phoneLineEdit"/>
                </item>
                <item row="4" column="0">
                 <widget class="QLabel" name="emailLabel">
                  <property name="text">
                   <string>Email</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="1">
                 <widget class="QLineEdit" name="emailLineEdit"/>
                </item>
                <item row="5" column="0">
                 <widget class="QLabel" name="addressLabel">
                  <property name="text">
                   <string>Address</string>
                  </property>
                 </widget>
                </item>
                <item row="5" column="1">
                 <widget class="QLineEdit" name="addressLineEdit"/>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="errorLabel">
               <property name="styleSheet">
                <string notr="true">QLabel{
	font: 700 12pt &quot;Segoe UI&quot;;
	color: rgb(255, 0, 0);
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="registerButtonLayout" native="true">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>50</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>50</height>
                </size>
               </property>
               <layout class="QHBoxLayout" name="buttonLayout">
                <property name="spacing">
                 <number>25</number>
                </property>
                <property name="leftMargin">
                 <number>200</number>
                </property>
                <property name="rightMargin">
                 <number>200</number>
                </property>
                <item>
                 <widget class="QPushButton" name="submitButton">
                  <property name="maximumSize">
                   <size>
                    <width>200</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Submit</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="cancelButton">
                  <property name="maximumSize">
                   <size>
                    <width>200</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Cancel</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QWidget" name="InstructionsLayout" native="true">
            <property name="styleSheet">
             <string notr="true">QLabel{
	font: 12pt &quot;Segoe UI&quot;;
}</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="spacing">
              <number>2</number>
             </property>
             <item>
              <widget class="QLabel" name="startedLabel">
               <property name="styleSheet">
                <string notr="true">#startedLabel{
	font: 700 12pt &quot;Segoe UI&quot;;
}</string>
               </property>
               <property name="text">
                <string>Getting Started</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="fillLabel">
               <property name="text">
                <string>Fill in this form to begin your membership</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="spacerLabel">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="needLabel">
               <property name="styleSheet">
                <string notr="true">#needLabel{
	font: 700 12pt &quot;Segoe UI&quot;;
}</string>
               </property>
               <property name="text">
                <string>You'll need three things:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="needdLabel">
               <property name="text">
                <string>- A Valid Form of ID</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="needEmailLabel">
               <property name="text">
                <string>- A Valid Email Address</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="needAdressLabel">
               <property name="text">
                <string>- A Physical City Address</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QWidget" name="footerBackground" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>110</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>110</height>
          </size>
         </property>
         <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0">
          <item row="0" column="2">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="footerLogo">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="images.qrc">:/branding/images/brand/logoSmall.png</pixmap>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>472</width>
              <height>5</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
